lib/kino/data_table.ex

```diff
@@ -212,7 +226,7 @@ defmodule Kino.DataTable do
     |> Enum.map(fn {_, idx} -> key_to_column(idx) end)
   end
 
-  defp columns_structure_for_record(record) when is_map(record) do
+  defp columns_structure_for_record(record) when is_map(record) or is_struct(record) do
```

Comment:
`is_map(record)` will match all structs already:

```suggestion
  defp columns_structure_for_record(record) when is_map(record) do
```